src/545.json

```diff
@@ -344,7 +344,7 @@
    "header" : [
       "Hi everyone!",
-      "Let me start by welcoming you to the new year! Let me also use the opportunity to thank <a href=\"http://manwar.org/\">Mohammad S. Anwar</a> for his dedication in both editing every other edition of the Perl Weekly Newsletter and his very own <a href=\"https://theweeklychallenge.org/\">Weekly Challenge</a>. Let me also thank <a href=\"https://davecross.co.uk/\">Dave Cross</a> for his immense help making our grammar a bit more bearable and the various Open Source projects he creates. (e.g. the <a href=\"https://cpandashboard.com/\">CPAN Dashboard</a> or <a href=\"https://perl.theplanetarium.org/\">The Perl Planetarium</a>). If you also like their work please thank them in an email, a blog post, or by supporting them via <a href=\"https://www.patreon.com/manwar/\">Patreon for Mohammad</a> or via <a href=\"https://github.com/davorg/\">GitHub for Dave</a>. They won't become wealthy from this, but receiving a paycheck every month from tens of supporters give a huge boost to the ego and makes one create even more for the public.",
+      "Let me start by welcoming you to the new year! Let me also use the opportunity to thank <a href=\"http://manwar.org/\">Mohammad S. Anwar</a> for his dedication in both editing every other edition of the Perl Weekly Newsletter and his very own <a href=\"https://theweeklychallenge.org/\">Weekly Challenge</a>. Let me also thank <a href=\"https://davecross.co.uk/\">Dave Cross</a> for his immense help making our grammar a bit more bearable and the various Open Source projects he creates. (e.g. the <a href=\"https://cpandashboard.com/\">CPAN Dashboard</a> or <a href=\"https://perl.theplanetarium.org/\">The Perl Planetarium</a>). If you also like their work please thank them in an email, a blog post, or by supporting them via <a href=\"https://www.patreon.com/manwar/\">Patreon for Mohammad</a> or via <a href=\"https://github.com/davorg/\">GitHub for Dave</a>. They won't become wealthy from this, but receiving a paycheck every month from tens of supporters give a huge boost to the ago and makes one create even more for the public.",
       "As for me, you might recall I've been learning Spanish for more than 3 years and started to learn Ladino (aka. Judeo-Spanish) a few weeks ago. Learning a foreign language is a totally different experience than learning a programming language. If for nothing else, because you don't get immediate feedback to your mistakes from a compiler. Well. Except if you use an application that helps you. My open source mindset quickly kicked in. I found an Open Source platform called <a href=\"https://librelingo.app/\">LibreLingo</a> that will help you learn languages. It is still in very early version, but started to use that platform to be able to practice Ladino and also started to contribute to the project. I'll share more about it when enough progress was made to show something in public. Until then you are invited to check it out.",
```

Comment:
Was this change ("ego" => "ago") intentional or some left over of last week where I fixed this too late. I think it should be "ego" not?

Reply:
No, "ego" is correct. I missed it when dealing with the merge conflict. Sorry.

Reply:
:+1: np